brave/brave-core
# Create BAT notification service to deliver BAT-related notifications

Fixes brave/brave-browser#922

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (`npm test brave_unit_tests && npm test brave_browser_tests`) on

## Test Plan:
- clean profile
- enable rewards
- click claim on grant
- click on BAT icon in the url bar
- make sure that you see notification about grant
- clicking on X should remove this notifiation

## Reviewer Checklist:
